### PR TITLE
[Fix][Zeta] Set `AsyncLogger.ThreadNameStrategy=UNCACHED` to avoid thread name cache

### PR DESCRIPTION
**Pull request by Hisoka-X**
### Purpose of this pull request
Force set `AsyncLogger.ThreadNameStrategy=UNCACHED` to avoid thread name cache. Thread name cache will cause task log prefix thread name not match job id. Please refer https://issues.apache.org/jira/browse/LOG4J2-2052.

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
tested in local

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
cc @hailin0

File: config/jvm_client_options [hunk 1]
```diff
@@ -19,6 +19,9 @@
 -Xms256m
 -Xmx512m
 
+# Fix LOG4J2-2052
+-DAsyncLogger.ThreadNameStrategy=UNCACHED
```
Comment (hailin0) [inline config/jvm_client_options]: move to here？
https://github.com/apache/seatunnel/blob/ca94e3beb0f4b5325e781839c2d3abb7659578da/seatunnel-core/seatunnel-starter/src/main/bin/seatunnel-cluster.sh#L86

Comment (Hisoka-X) [reply config/jvm_client_options]: done.

**Review by hailin0 — COMMENTED** [review]
LGTM

waiting for ci passed

File: pom.xml [hunk 2]
```diff
@@ -103,7 +103,7 @@
         <elasticsearch6.client.version>6.3.1</elasticsearch6.client.version>
         <elasticsearch7.client.version>7.5.1</elasticsearch7.client.version>
         <flink-shaded-hadoop-2.version>2.7.5-7.0</flink-shaded-hadoop-2.version>
-        <commons-lang3.version>3.5</commons-lang3.version>
+        <commons-lang3.version>3.8</commons-lang3.version>
```
Comment (Hisoka-X) [inline pom.xml:106]: fix https://issues.apache.org/jira/browse/LANG-1384

**Review by dailai — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**